config/routes.rb

```diff
@@ -6,7 +6,7 @@
   devise_scope :user do
     get 'signin', to: 'devise/sessions#new' 
     delete 'users/sign_out', to: 'devise/sessions#destroy'
-    root to: 'dashboard#index'
+    root to: 'dashboard#index"
```

Comment:
I think this change crashed the project index page.

Reply:
yes now working fine